src/components/ReportActionItem/ReportPreview.js

```diff
@@ -100,6 +101,31 @@ const propTypes = {
     /** Whether a message is a whisper */
     isWhisper: PropTypes.bool,
 
+    /** All of the transaction violations */
+    transactionViolations: PropTypes.shape({
```

Comment:
As this is used multiple times, create new file in in components/ (transactionViolationsPropTypes.js)